bower.json

```diff
@@ -0,0 +1,24 @@
+{
+  name: "ggoodman-angular-drag-drop",
```

Comment:
This file does not appear to be valid json (unquoted keys). Does bower support pseudo-json files?